rbac/api/cross_access/view.py

```diff
@@ -201,3 +201,13 @@ def validate_and_get_input_for_creation(self, request_data):
         request_data["end_date"] = end_date
         request_data["user_id"] = self.request.user.user_id
         request_data["roles"] = [{"display_name": role} for role in roles]
+
+    def create_principal(self, target_account, user_id):
+        """Create a cross account principal in the target account."""
+        # Principal would have the pattern acctxxx-123456.
+        principal_name = f"{target_account}-{user_id}"
+        tenant_schema = create_schema_name(target_account)
```

Comment:
As an aside, we should probably rename this at some point (not now). `create_schema_name` is a bit confusing, because at a glance it always makes me think we're creating a schema. :)

Reply:
Ah, yea, right. That makes confusion. get_schema_from_account or sth similar feels better 👍